client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/client/binding/NoBindingSuggestionFoundParams.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.clientapi.client.binding;
+
+import org.eclipse.lsp4j.jsonrpc.validation.NonNull;
+
+public class NoBindingSuggestionFoundParams {
+
+  @NonNull
+  private final String projectKey;
+
+  public NoBindingSuggestionFoundParams(String projectKey) {
```

Comment:
We could also send the text to display. The only unique thing is the action for showing the troubleshooting documentation, it differs for each IDE.